boa_engine/src/vm/code_block.rs

```diff
@@ -458,6 +465,68 @@ impl JsVmFunction {
     }
 }
 
+#[derive(Debug)]
+#[allow(missing_copy_implementations)]
+pub(crate) struct JsVmGeneratorFunction;
```

Comment:
Why are we allowing missing copy implementations? This could easily implement `Copy`, right?

Also, we should add some documentation to it (and its methods) to properly understand what's about.

Reply:
This struct was only used in one place. I replaced it with a function, as that makes more sense here.